lib/request.js

```diff
@@ -5,10 +5,35 @@
  */
 
 const logger = require('debug')('node-bigcommerce:request'),
-  https = require('https');
+  https = require('https'),
+  zlib = require('zlib');
 
 const version = require('../package.json').version;
 
+/**
+ * Parse response
+ */
+function parseResponse(res, body, resolve, reject) {
+  try {
+    if (!/application\/json/.test(res.headers['content-type']) || body.trim() === '') {
+      return resolve(body);
+    }
+
+    const json = JSON.parse(body);
+
+    if (Object.prototype.hasOwnProperty.call(json, 'error') || Object.prototype.hasOwnProperty.call(json, 'errors')) {
```

Comment:
Since this is a plain parsed object, why not check for properties directly? i.e. `if (json.error || json.errors) {`

Reply:
Hey, great question. I believe that's how it is in the code currently, that particular piece was just moved. I can change the style if you'd like, no problem. Let me know.

Reply:
No problem! Thanks! Done.